architecture/core_objects/overview.adoc

```diff
@@ -5,7 +5,20 @@
 :icons:
 :experimental:
 
-If you'd like to contribute to OpenShift documentation, see our
-https://github.com/openshift/openshift-docs[source repository] and
-https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc[guidelines]
-to get started.
+The following sections give some information about general OpenShift and
+Kubernetes architecture and terms. The
+link:../architecture/core_objects/kubernetes_model.html[Kubernetes Model]
+section outlines how the Kubernetes infrastructure is used in your OpenShift
+instance. The link:../architecture/core_objects/openshift-model.html[OpenShift
+Model] section outlines any OpenShift-specific information about your
+infrastructure.
```

Comment:
Thanks for filling this stub out. It's a bit of a misnomer to say the `kubernetes_model` topic outlines the "Kubernetes infrastructure", when in a neighboring directory we have `infrastructure_components/kubernetes_infrastructure`. Maybe something like:

"The following topics provide information on important resources in the link:kubernetes_model.html[Kubernetes model], which is extended by the link:openshift_model[OpenShift model]."